acquire/acquire.py

```diff
@@ -2251,6 +2251,12 @@ def main() -> None:
                 target_name = target.name
                 log.info("Loading target %s", target_name)
                 log.info(target)
+                # Check if we are just listing children and not collecting
+                if args.list_children:
+                    log.info(f"Listing children on Target: %s", target.name)
+                    for index, child in enumerate(target.list_children()):
+                        log.info(f"- Found child #%s: type=%s, path=%s", index, child.type, child.path)
```

Comment:
```suggestion
                        log.info("- [#%s]: type=%s, path=%s", index, child.type, child.path)
```
Dunno if this looks any nicer?